packages/client/src/intros/ZonweringRolluikIntro.tsx

```diff
@@ -1,20 +1,21 @@
 import React from "react";
+import { useTranslation } from "react-i18next";
 
 import { Intro } from "./shared";
 
-export default () => (
-  <Intro
-    usableForBullets={[
-      "het wijzigen van een monument.",
-      "het plaatsen van de zonwering, het rolhek, rolluik of luik zelf.",
-    ]}
-    usableForText={`U kunt deze vergunningcheck gebruiken als u een nieuwe
-      zonwering, rolhek, rolluik of luik plaatst of als u een bestaande
-      vernieuwt.`}
-    exceptions={[
-      "Het aantal woningen verandert.",
-      `Het gebouw, of het deel van het gebouw waarin de zonwering, het rolhek,
-        rolluik of luik komt, is zonder vergunning gebouwd.`,
-    ]}
-  />
-);
+export default () => {
+  const { t } = useTranslation();
+  return (
+    <Intro
+      usableForBullets={[
+        t("introPage.common.monument bullet"),
+        t("introPage.zonwering.placing bullet"),
+      ]}
+      usableForText={t("introPage.zonwering.intro description")}
+      exceptions={[
+        t("introPage.common.amount of houses exception"),
+        t("introPage.zonwering.build without permit exception"),
```

Comment:
Seeing this I'm now wondering why we use the dutch word "zonwering" and not "blinds" instead. Maybe we can keep the language file's keys "introPage.zonwering.build..." all in English "introPage.blinds.build...". The same goes for the name of this file.

What do you think?

Reply:
I totally agree!